lms/envs/common.py

```diff
@@ -191,13 +191,13 @@
     'RESTRICT_AUTOMATIC_AUTH': True,
 
     # Toggle the availability of the shopping cart page
-    'ENABLE_SHOPPING_CART': False,
+    'ENABLE_SHOPPING_CART': True,
```

Comment:
This can and should be enabled in configuration repo for this project.
Please remove and add comment/PR to the configuration.

Reply:
Removed. And added a comment to the PR.